### ISSUE TITLE
Offset Attribute is Not Applied in Live Preview

### ISSUE THREAD

**Issue by kmgdevelopment** [engram-design/SuperTable#208]
### Description
When the `offset` attribute is used on a Super Table field, the offset number is not applied in the Live Preview. Example:

````
{% for item in entry.newsItems.offset(1).all %}
    {{ item.title }}<br>
{% endfor %}
````

This correctly omits the first news item in the front-facing template, but does not omit the first item in the Live Preview.

### Steps to reproduce

1. Use the `offset` attribute in a Super Table looping tag
2. View the template using Live Preview

### Additional info

- Plugin version: 2.0.10
- Craft version: 3.0.27.1

**Comment by engram-design** [comment]
I've noticed a similar issue for Matrix actually: https://github.com/craftcms/cms/issues/2801 that's still unresolved. Will investigate.

**Comment by kmgdevelopment** [comment]
How're we doing on this?

**Comment by engram-design** [comment]
Pretty sure this is a bigger issue than Super Table. So far, this is our best answer - https://github.com/craftcms/cms/issues/2801#issuecomment-437118428

**Comment by engram-design** [comment]
FYI, this has been resolved for the next 3.2 Alpha release! https://github.com/craftcms/cms/issues/1787#issuecomment-493640314

**Comment by engram-design** [comment]
This should be fixed in Craft 3.2.
